### PR TITLE
[DOCS] Remove initialisation of contract with wei FAQ item

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Description

I feel like this FAQ item is covered in _control-structures.rst_ line 73, but the example doesn't mention the `new` syntax, so maybe I am wrong, in which case I'll add this into that section.

Part of https://github.com/ethereum/solidity/issues/1219

### Checklist
- [x] Code compiles correctly
- [x] All tests are passing
- [x] New tests have been created which fail without the change (if possible)
- [x] README / documentation was extended, if necessary
- [x] Changelog entry (if change is visible to the user)
- [x] Used meaningful commit messages

### CODE REVIEW

**Comment by erak** [comment]
I think it's fine to remove that FAQ item since it's covered in the paragraph you mentioned and also in the next section: https://solidity.readthedocs.io/en/v0.5.3/control-structures.html#creating-contracts-via-new.

But: this PR needs to be rebased ;)

File: docs/frequently-asked-questions.rst [hunk 1]
```diff
@@ -2,37 +2,5 @@
...
-        function test() public {
-            child = (new B).value(10)(); //construct a new B with 10 wei
-        }
-    }
-
-More Questions?
-===============
-
-If you have more questions or your question is not answered here, please talk to us on
+If your question is not answered here, please talk to us on
```
Comment (erak) [inline docs/frequently-asked-questions.rst:5]: Since this was the last item, I decided to not delete the FAQ completely, but rather leave that comment.

**Review by erak — COMMENTED** [review]
Should we delete the FAQs now completely or is it fine to leave a note about our dev channel / the issue tracker?

**Review by chriseth — APPROVED** [review]

**Comment by chriseth** [comment]
Thanks for your great contributions, @cyrusadkisson !

**Comment by ChrisChinchilla** [comment]
@bit-shift and @chriseth I think we delete it, I'll create a new PR asap.

**Pull request merged**